providers/common/ciphers/cipher_gcm.c

```diff
@@ -138,7 +186,13 @@ int gcm_get_ctx_params(void *vctx, OSSL_PARAM params[])
             return 0;
         }
     }
-
+    p = OSSL_PARAM_locate(params, OSSL_CIPHER_PARAM_GCM_GET_IV_GEN);
+    if (p != NULL) {
+        if (p->data == NULL
+            || p->data_type != OSSL_PARAM_OCTET_STRING
+            || !getivgen(ctx, p->data, p->data_size))
```

Comment:
@levitte - is there a better way to do this?

Reply:
Not that I can think of for the moment...